applications/luci-app-mwan3/luasrc/model/cbi/mwan/ruleconfig.lua

```diff
@@ -15,62 +14,72 @@ local ipsets = util.split(util.trim(util.exec("ipset -n -L 2>/dev/null | grep -v
 m = Map("mwan3", translatef("MWAN Rule Configuration - %s", arg[1]))
 m.redirect = dsp.build_url("admin", "network", "mwan", "rule")
 
-mwan_rule = m:section(NamedSection, arg[1], "rule", "")
-mwan_rule.addremove = false
-mwan_rule.dynamic = false
+s = m:section(NamedSection, arg[1], "rule", "")
+s.addremove = false
+s.dynamic = false
 
-src_ip = mwan_rule:option(Value, "src_ip", translate("Source address"),
+o = s:option(ListValue, "family", translate("Internet Protocol"))
+o.default = "ipv4"
```

Comment:
Default should be "any" since that is the current default, and this could start switching rules to ipv4 if no option is explicitly set.

Reply:
yes default should be "ipv4 + ipv6", but please never use "any" as value and save to config file, we never use "any"
we just leave family to `""` do indicate "any"

Reply:
Yes that is true with any and it should not be only ipv4. Thanks.
But I think the default should not be **""** it would be better to set the value to **any**.
In mwan3 the option family is also set to [any](https://github.com/openwrt/packages/blob/master/net/mwan3/files/lib/mwan3/mwan3.sh#L950) if this option is not in the configured